### PR TITLE
Feature/gcb 177 add error view to watch widget

### PR DESCRIPTION
**Pull request by michallaskowski**
Make sure these boxes are checked before submitting your Pull Request - thank you!

- [x] [SwiftLint](https://github.com/Realm/SwiftLint) is installed on your system
- [x] The project does not generate any warnings or errors
- [x] The tests are passing
- [x] Branch name is in line with [GitFlow](http://jeffkreeftmeijer.com/2010/why-arent-you-using-git-flow/)

### CODE REVIEW

File: GrandCentralBoard/Widgets/Watch/View/WatchWidgetView.swift [hunk 1]
```diff
@@ -14,6 +14,7 @@ final class WatchWidgetView: UIView, ViewModelRendering {
     @IBOutlet private weak var alignedTimeLabel: UILabel!
     @IBOutlet private weak var eventLabel: UILabel!
 
+    private weak var failureView: UIView?
```
Comment (krris) [inline GrandCentralBoard/Widgets/Watch/View/WatchWidgetView.swift]: `failureView` is never initialized.

File: GrandCentralBoard/Widgets/Watch/View/WatchWidgetView.swift [hunk 2]
```diff
@@ -46,11 +47,14 @@ final class WatchWidgetView: UIView, ViewModelRendering {
...
-                setUpLabelsWithViewModel(viewModel)
-                setUpImagesWithViewModel(viewModel)
-            default:
-                break
+        case (_, .Rendering(let viewModel)):
+            removeFailureView()
+            setUpLabelsWithViewModel(viewModel)
+            setUpImagesWithViewModel(viewModel)
+        case (_, .Failed):
+            showFailureView()
```
Comment (krris) [inline GrandCentralBoard/Widgets/Watch/View/WatchWidgetView.swift]: If widget is failing for a long time `fillViewWithView` can be called multiple times. It leads to situation where `failureView` is added multiple times as a subview.

**Comment by krris** [comment]
Should we display an error if there are no meetings in calendar? 

Current state of `develop`:
screen shot 2016-05-20 at 10 19 21
This branch:
screen shot 2016-05-20 at 10 17 54

File: GrandCentralBoard/Widgets/Watch/View/WatchWidgetView.swift (hunk 2 above)
Comment (michallaskowski) [inline GrandCentralBoard/Widgets/Watch/View/WatchWidgetView.swift]: Good catch

**Comment by michallaskowski** [comment]
It fails because the widget tries to fetch data from localhost.

**Pull request merged**